taccsite_cms/static/site_cms/js/modules/setTargetForExternalLinks.js

```diff
@@ -1,37 +1,4 @@
-/**
- * Whether to log debug info to console
- * @const {string}
- */
-const SHOULD_DEBUG = window.DEBUG;
```

Comment:
**Woah. Control the burn.** The `SHOULD_DEBUG` is still used. Also, the [jsDoc block](https://jsdoc.app/about-block-inline-tags) (a [docblock](https://en.wikipedia.org/wiki/Docblock)) is still accurate.

Reply:
Sorry. https://apnews.com/article/texas-panhandle-fire-evacuations-cbbb6a279bef1bd020722ed48927114a

Reply:
Restored via [cbad538](https://github.com/TACC/Core-CMS/pull/805/commits/cbad538bc59e333040f84fd3e665cb388c71b5af)